### DISCUSSION TITLE
MOOSE website not working

### DISCUSSION THREAD

**Discussion by Samin-Moghimiasl** [idaholab/moose#21686] [Q&A General] [answered]
Hi all, 

MOOSE website has been down for a couple of hours now. Is anyone else having the same issues? Any ideas of when this can be resolved?

Many thanks, 
Samin

**Comment by GiudGiud** [comment]
Hello

We are aware of this, it's due to testing infrastructure migration. We are working with INL infrastructure to bring it back online. It should be back now or very soon
EDIT: should be working now

Guillaume

**Comment by cticenhour** [comment, marked as answer]
The migration was completed, and https://mooseframework.inl.gov/ is back online.

**Comment by WilkAndy** [comment]
Hmmm, i still can't connect, and my PRs isn't testing properly either.  Do i have to do something special, eg, to activate the code coverage on PRs?  Other PRs, eg, https://github.com/idaholab/moose/pull/21699 , seem to work OK.

**Reply by cticenhour** [reply]
Looks like https://civet.inl.gov is down again, but was working earlier. So that's why the webhook governing the CI connection isn't working for PRs right now. 

Tagging @loganharbour @milljm

**Reply by loganharbour** [reply]
When civet was down, the hook from GitHub that told it a PR was created/updated would not complete. This was unfortunately intentional as we were transitioning to a new server and we had to pause all writes to history on civet. 

The easiest way to force a new hook for a PR is to rewrite history for the last commit:

```
git commit --amend --no-edit
git push --force
```

this will basically edit the last commit, changing its date to now. With “new” commits, a push will trigger a new event.

**Reply by WilkAndy** [reply]
Thank you @loganharbour and @cticenhour .  I bet when i wake up tomorrow everything will be hunky dorey.

**Reply by milljm** [reply]
Still down. Wonder what happened... Ticket submitted.
